src/htmx.js

```diff
@@ -1432,7 +1433,7 @@ var htmx = (function() {
       swapStyle = oobValue
     }
 
-    const targets = getDocument().querySelectorAll(selector)
+    const targets = querySelectorAllExt(rootNode, selector, false)
```

Comment:
Taking a second look at this, since `oobSwap` is exposed in the internal API, itself exposed to extensions:
https://github.com/bigskysoftware/htmx/blob/0023cd85f01987dfa0add4101e08d98c7801d88e/src/htmx.js#L326

Maybe we should add a `|| getDocument()` fallback here ? In case that new parameter isn't set, which would be the case for any existing extension that used this method